llvm/test/CodeGen/AArch64/fold-int-pow2-with-fmul-or-fdiv.ll

```diff
@@ -0,0 +1,444 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -mtriple=aarch64-none-linux-gnu -mattr=+neon | FileCheck %s --check-prefixes=CHECK,CHECK-NEON
```

Comment:
+neon should be the default, so for no-neon it likely need -mattr=-neon.
That is if you feel you need to test without neon, it might not be necessary?